rules/jsonschema-2019-09-to-2020-12.json

```
@@ -10,7 +10,8 @@
         "required": [ "$schema" ],
         "properties": {
           "$schema": {
-            "const": "https://json-schema.org/draft/2019-09/schema"
+            "type":"string",
```

```suggestion
            "type": "string",
```

Let's be consistent with spacing after `:` here and in the other rules